pkg/raft/raft.go

```diff
@@ -345,6 +345,8 @@ type raft struct {
 
 	// the leader id
 	lead uint64
+	// the last accepted term
+	accTerm uint64
```

Comment:
We should initialize `accTerm` to be last entry term, somewhere in `newRaft` func. If we don't, then a restarted node won't be able to advance commit index immediately, it will need an append message first, to get `appTerm` up-to-date.

It would also be good to document the invariant (see above).